GSA-TTS/identity-site
# LG-12841: Remove security practices in favor of policy page

## 🎫 Ticket

[LG-12841](https://cm-jira.usa.gov/browse/LG-12841)

## 🛠 Summary of changes

This removes the "Our Security Practices" (`/policy/our-security-practices/`) page, substituting the top-level "Policy" (`/policy/`) to serve in its place.

It includes:

- Updating existing internal links to point to `/policy/`
- Setting up a redirect from `/policy/our-security-practices/` to `/policy/`
- Removing the "Our Security Policy" page and navigation links
- Avoiding redirect loop from "Policy" to "Our Security Practices" page
- Applying all of the above to all translated versions of the site

## 📜 Testing Plan

1. Go to https://federalist-17bd62cc-77b7-4687-9c62-39b462ce6fd5.sites.pages.cloud.gov/preview/gsa-tts/identity-site/aduth-lg-12841-rm-security-practices/
2. Verify that you cannot access "Our Security Practices" page
   1. Side navigation in Policy does not include a link
   2. References to security practices in existing page content link to "Policy" page
